src/main/java/de/muenchen/isi/infrastructure/entity/stammdaten/SobonJahr.java

```diff
@@ -0,0 +1,30 @@
+package de.muenchen.isi.infrastructure.entity.stammdaten;
+
+import de.muenchen.isi.infrastructure.entity.BaseEntity;
+import java.time.LocalDate;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Entity
+@Data
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class SobonJahr extends BaseEntity {
+
+    @Column(nullable = false)
+    private Integer jahr; // JJJJ
+
+    @Column(nullable = false)
+    private LocalDate gueltigAb;
+
+    @ElementCollection
+    private List<StaedtbaulicherOrientierungwert> staedtebaulicheOrientierungswerte;
```

Comment:
Die Listen-Attribute `staedtebaulicheOrientierungswerte` und `sobonOrientierungswerte` sollten jeweils in eine eigene Klasse mit dem Attribut `gueltigAb` ausgelagert werden.

Somit ist gewährleistet, dass die Stammdaten der `StaedtebaulicheOrientierungswerte` und `SobonOrientierungswerte` bei Bedarf unterschiedliche gueltigAb-Werte besitzen darf.